packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html

```diff
@@ -1,4 +1,7 @@
-<oui-spinner data-size="s" data-ng-if="$ctrl.isLoading"></oui-spinner>
+<oui-spinner
+    data-size="s"
+    data-ng-if="$ctrl.hasOwnProperty('displayLoader') ? ($ctrl.isLoading && $ctrl.displayLoader) : $ctrl.isLoading"
```

Comment:
in our case if we use data-ng-if="($ctrl.displayLoader && $ctrl.isLoading) || $ctrl.isLoading" the spinner will always appear  because even if we have $ctrl.displayLoader set to false we will check $ctrl.isLoading witch is true so we will get a wrong behaviour